plugins/doc_fragments/foreman.py

```diff
@@ -270,6 +270,13 @@ class ModuleDocFragment(object):
       - Only available for Katello installations.
     required: false
     type: str
+  activation_keys:
+    description:
+      - Activation Keys used for deployment.
+      - Coma separated list.
```

Comment:
Let's put a little consciousness in that list.
```suggestion
      - Comma separated list.
```